docs/commands/zq.md

```diff
@@ -582,7 +582,7 @@ Next, a JSON file can be converted from ZNG using:
 zq -f json conn.zng > conn.json
 ```
 Note here that we lose information in this conversion because the rich data types
-of Zed (that were [translated from the Zeek format](../../zeek/Data-Type-Compatibility.md)) are lost.
+of Zed (that were [translated from the Zeek format](https://github.com/brimdata/zed/blob/main/zeek/Data-Type-Compatibility.md)) are lost.
```

Comment:
These kinds of relative links to things outside the `/docs` directory work fine when browsing within GitHub itself, but they understandably break on the deployed docs site. I've therefore fixed them by using an absolute URL.